### PR TITLE
[CROSSDATA-305] [CORE] Implement persistence layer (Zookeeper)

### PR DESCRIPTION
**Pull request by jjlopezm**
Zookeeper views implementation.

### CODE REVIEW

File: core/src/main/scala/org/apache/spark/sql/crossdata/catalog/ZookeeperCatalog.scala [hunk 1]
```diff
@@ -13,30 +13,46 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+/**
```
Comment (miguel0afd) [inline core/src/main/scala/org/apache/spark/sql/crossdata/catalog/ZookeeperCatalog.scala]: Looks like this file contains two headers of the same license

File: core/src/main/scala/org/apache/spark/sql/crossdata/catalog/ZookeeperCatalog.scala [hunk 2]
```diff
@@ -91,13 +106,35 @@ class ZookeeperCatalog(override val conf: CatalystConf = new SimpleCatalystConf(
...
+  override protected def lookupView(tableName: String, databaseName: Option[String]): Option[String] = {
+    if (viewDAO.dao.count > 0) {
+      val findView = viewDAO.dao.getAll()
+        .find(viewModel => viewModel.name == tableName && viewModel.database == databaseName)
+
+      findView match {
+        case Some(zkView) =>
+          Some(zkView.sqlViewField)
+        case None =>
+          viewDAO.logger.warn("View not exists")
```
Comment (miguel0afd) [inline core/src/main/scala/org/apache/spark/sql/crossdata/catalog/ZookeeperCatalog.scala]: Better with "View doesn't exist", isn't it?

File: core/src/main/scala/org/apache/spark/sql/crossdata/catalog/ZookeeperCatalog.scala [hunk 3]
```diff
@@ -91,13 +106,35 @@ class ZookeeperCatalog(override val conf: CatalystConf = new SimpleCatalystConf(
...
+          None
+      }
+    } else {
+      viewDAO.logger.warn("View path not exists")
```
Comment (miguel0afd) [inline core/src/main/scala/org/apache/spark/sql/crossdata/catalog/ZookeeperCatalog.scala]: Better with "View path doesn't exist", isn't it?

File: server/pom.xml [hunk 4]
```diff
@@ -318,10 +318,10 @@
                                     <from>target/crossdata-dist-${project.version}/conf/</from>
                                     <to>/etc/sds/crossdata</to>
                                 </copyDirectory>
-                                <copyDirectory>
-                                    <from>target/crossdata-dist-${project.version}/lib/</from>
-                                    <to>/opt/sds/crossdata/lib</to>
-                                </copyDirectory>
+                                <!--<copyDirectory>-->
```
Comment (miguel0afd) [inline server/pom.xml]: Could this comment be removed?

File: server/src/main/scala/com/stratio/crossdata/server/config/NumberActorConfig.scala [hunk 5]
```diff
@@ -13,21 +13,39 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+/**
```
Comment (miguel0afd) [inline server/src/main/scala/com/stratio/crossdata/server/config/NumberActorConfig.scala]: Again, two license headers in the same file

**Comment by miguel0afd** [comment]
:+1:

**Comment by compae** [comment]
LGTM

**Pull request merged**